From d9db29db567012b7c72e85e1be1fbf55fcc9b667 Mon Sep 17 00:00:00 2001
From: Niels De Graef <nielsdegraef@gmail.com>
Date: Sat, 11 May 2019 09:02:34 +0200
Subject: [PATCH] gkr: Use 0 on empty flags

A Flags-type variable without any flag set can be replaced with 0, so
this is a safe thing to do. It also prevents us from having to deal with
the accidental API break in libsecret (see
https://gitlab.gnome.org/GNOME/libsecret/merge_requests/19)
---
 gkr/gkr-keyring-add.vala | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/gkr/gkr-keyring-add.vala b/gkr/gkr-keyring-add.vala
index 4e92a520..f60c9a22 100644
--- a/gkr/gkr-keyring-add.vala
+++ b/gkr/gkr-keyring-add.vala
@@ -41,8 +41,7 @@ public class Seahorse.Gkr.KeyringAdd : Gtk.Dialog {
 
         var cancellable = Dialog.begin_request(this);
         var service = Backend.instance().service;
-        Secret.Collection.create.begin(service, this.name_entry.text, null,
-                                       Secret.CollectionCreateFlags.COLLECTION_CREATE_NONE,
+        Secret.Collection.create.begin(service, this.name_entry.text, null, 0,
                                        cancellable, (obj, res) => {
             /* Clear the operation without cancelling it since it is complete */
             Dialog.complete_request(this, false);
-- 
2.20.1

